http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -178,18 +182,18 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
 
       clusterId = saveClusterResult.minimalCluster.id
 
-      machineConfigFromReqAndConfig = req.kubernetesRuntimeConfig.getOrElse(
+      machineConfig = req.kubernetesRuntimeConfig.getOrElse(
         KubernetesRuntimeConfig(
           config.leoKubernetesConfig.nodepoolConfig.galaxyNodepoolConfig.numNodes,
           config.leoKubernetesConfig.nodepoolConfig.galaxyNodepoolConfig.machineType,
-          config.leoKubernetesConfig.nodepoolConfig.galaxyNodepoolConfig.autoscalingEnabled
+          numOfReplicas.fold(
+            config.leoKubernetesConfig.nodepoolConfig.galaxyNodepoolConfig.autoscalingEnabled
+          )(_ =>
+            true
+          ) // if numOfReplicas is defined, then autoscaling has to be enabled because we scale up/down replicas to pause/resume apps.
```

Comment:
Gotcha, it should not impact things as they are until we enable autoscaling for Galaxy (AOU is not exposing this app correct?)

Reply:
yep...this doesn't affect Galaxy, and AoU is not using Galaxy